src/components/rate/index.tsx

```diff
@@ -8,5 +8,12 @@ interface RatingProps {
 }
 
 export function Rating({ defaultValue = 0, onChange }: RatingProps) {
-  return <Rate defaultValue={defaultValue} onChange={onChange}  />;
+  return (
+    <Rate
+      size="xs"
+      style={{ width: "6rem" }}
+      defaultValue={defaultValue}
+      onChange={onChange}

```

Comment:
style in-line nao é mt bacana

Reply:
não achei outro jeito de da um gap nas estrelas nessa lib

Reply:
extrai pra um componente styled